dnf-behave-tests/features/plugins-core/needs-restarting-conf-files.feature

```diff
@@ -0,0 +1,122 @@
+Feature: Add package to needs-restarting using config files
+
+Background:
+    Given I enable plugin "needs_restarting"
+      And I use repository "dnf-ci-fedora"
+      And I move the clock backward to "before boot-up"
+      And I execute dnf with args "install lame kernel basesystem glibc wget abcde"
+      And I move the clock forward to "the present"
+      And I use repository "dnf-ci-fedora-updates"
+    Given I create directory "/etc/dnf/plugins/needs-restarting.d"
+      And I create and substitute file "/etc/dnf/plugins/needs-restarting.d/wget.conf" with
+      """
+      wget
+      """
+      Given I do not set config file
```

Comment:
Almost forgot, please rebase, this step was removed. Not sure why you have it here exactly but I think you just won't need it anymore (the config is now generated inside the installroot automatically).

Also, you're using multiple `Given`s here, you can just use `And` to be more consistent.